test/kiwixTitle.cpp

```diff
@@ -0,0 +1,45 @@
+#include "gtest/gtest.h"
+#include <string>
+#include "../src/tools/stringTools.h"
+
+TEST(KiwixTitleTest, ToTitleEmpty)
+{
+    ASSERT_EQ("", kiwix::toTitle(""));
+}
+
+TEST(KiwixTitleTest, ToTitleBasic)
+{
+    ASSERT_EQ("Hello", kiwix::toTitle("hello"));
+    ASSERT_EQ("Hello", kiwix::toTitle("HELLO"));
+    ASSERT_EQ("Hello", kiwix::toTitle("Hello"));
+}
+
+TEST(KiwixTitleTest, ToTitleMultiWord)
+{
+    ASSERT_EQ("Hello World", kiwix::toTitle("hello world"));
+    ASSERT_EQ("Hello World", kiwix::toTitle("HELLO WORLD"));
+    ASSERT_EQ("The Quick Brown Fox", kiwix::toTitle("the quick brown fox"));
+}
+
+TEST(KiwixTitleTest, ToTitleSpecialChars)
+{
+    ASSERT_EQ("Über", kiwix::toTitle("über"));
+    ASSERT_EQ("Étoile", kiwix::toTitle("étoile"));
+    ASSERT_EQ("Ñandú", kiwix::toTitle("ñandú"));
+    ASSERT_EQ("Français", kiwix::toTitle("français"));
+    ASSERT_EQ("Pусский", kiwix::toTitle("pусский"));
+}
+
+TEST(KiwixTitleTest, ToTitleMixedCase)
+{
+    ASSERT_EQ("Mixed Case", kiwix::toTitle("mIxEd cAsE"));
+    ASSERT_EQ("Camelcase", kiwix::toTitle("camelCase"));
+    ASSERT_EQ("Pascal Case", kiwix::toTitle("Pascal Case"));
+}
+
+TEST(KiwixTitleTest, ToTitleWithSymbols)
+{
+    ASSERT_EQ("Hello-World", kiwix::toTitle("hello-world"));
+    ASSERT_EQ("Don't Stop", kiwix::toTitle("don't stop"));
+    ASSERT_EQ("C++ Programming", kiwix::toTitle("c++ programming"));
+}
```

Comment:
Thanks for these tests!

However if we introduce a new unit-test .cpp file per function tested we may run out of disk space in our build tree (every unit test binary file links statically with `gtest` lib and is too large if it tests only one small utility). Hence, please move this tests into `test/stringTools.cpp`